template/makeBp.js

```diff
@@ -72,6 +72,8 @@ function getDataTypeSafeName(apiElement, attrName) {
     var pfTypeName = apiElement[attrName];
     if (pfTypeName === "SourceType") // In Unreal, the Enum ESourceType exists in the Android builder and conflicts with our ESourceType enum
         return "PfSourceType";
+    if (pfTypeName === "TriggerType")
```

Comment:
Approved after an additional comment here about why this replacement exists (mirror 2 lines up).